### PR TITLE
Update index code snippet.

### PR DESCRIPTION
**Pull request by 95-martin-orion**
CC @karlunho, @dstrain115

Discussion with users highlighted two issues with the previous code snippet:
- The 6x6 grid is too large for the average desktop (requires ~256GB RAM)
- Lack of measurements in the circuit triggered a Cirq error

My concerns with this:
- Will changing the snippet length mess up the [qsim landing page](https://quantumai.google/qsim)?
- Was there a reason we needed to mention multi-core Intel CPUs?

### CODE REVIEW

**Comment by dstrain115** [comment]
It's fine with me.  I think the snippet should be a "hello world" that's easy to run and works (not one that requires a multi-processor setup).

**Comment by 95-martin-orion** [comment]
+@lamberta for feedback on how this will affect the site. Will the code box resize automatically, or do we need to manually adjust it to fit the new snippet?

**Comment by lamberta** [comment]
The snippet box will automatically expand *vertically*. But the max width is capped so it will add a horizontal scrollbar if the content extends beyond the visible area.

We generally use these landing page snippets to illustrate very high-level usage or concepts—kind of a way to let the reader know what problems this library will help them solve and confirm they're in the right section of the website. Correct is good, but it's also fine to skip over setup that detracts from that high-level messaging. You can always link to a working Colab tutorial for that.

**Comment by 95-martin-orion** [comment]
Width shouldn't be an issue - the font is fixed-width and no lines were made longer by this change.

My reasoning for updating this is that a user was trying it for their first use of qsim. From their perspective that's a logical move, since it's the first code snippet they encounter on the site.

**Comment by 95-martin-orion** [comment]
+@sergeisakov for final review and merge.

File: docs/_index.yaml [hunk 1]
```diff
@@ -59,14 +59,18 @@ landing_page:
         import cirq
         import qsimcirq
 
-        # Pick up to ~36 qubits to run on a large multi-core Intel CPU machine
-        qubits = [cirq.GridQubit(i,j) for i in range(6) for j in range(6)]
+        # Pick up to ~25 qubits to simulate (requires ~128MB of RAM)
```
Comment (sergeisakov) [inline docs/_index.yaml:62]: It requires 256MB of RAM for 25 qubits.

**Review by sergeisakov — APPROVED** [review]

**Review by dstrain115 — APPROVED** [review]

**Pull request merged**